launch/pacmod_game_control.launch

```diff
@@ -28,7 +28,7 @@
        VEHICLE_6
        JUPITER_SPIRIT -->
   <!-- <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" /> -->
-   <arg name="pacmod_vehicle_type" default="VEHICLE_5" />
+   <arg name="pacmod_vehicle_type" default="JUPITER_SPIRIT" />
```

Comment:
Should this default be changed? also there is a dead line of code that should probably be removed? `<!-- <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" /> -->`

Reply:
yes, it was missed!